cfme/infrastructure/cluster.py

```diff
@@ -117,7 +117,12 @@ class All(CFMENavigateStep):
     prerequisite = NavigateToAttribute('appliance.server', 'LoggedIn')
 
     def step(self):
-        self.parent_view.navigation.select('Compute', 'Infrastructure', 'Clusters')
+        try:
```

Comment:
Did you navigate on a Cluster object or class? (message seems to indicate object). Its not optional, and the object attribute is set in __init__.... so that is strange.

That aside, I think this may warrant a separate navigation destination, 'Deployment Roles', to be used by Openstack clusters.  Otherwise this 'All' destination, which should be navigatable by the class, would now require an instantiated Cluster to navigate.